chromadb/auth/token_authn/__init__.py

```diff
@@ -71,9 +78,19 @@ def __init__(self, system: System) -> None:
         _check_token(self._token.get_secret_value())
 
         if system.settings.chroma_auth_token_transport_header:
-            self._token_transport_header = TokenTransportHeader[
-                str(system.settings.chroma_auth_token_transport_header)
-            ]
+            allowed_token_headers = [header.value for header in TokenTransportHeader]
+            if (
+                system.settings.chroma_auth_token_transport_header
+                not in allowed_token_headers
+            ):
+                raise ValueError(
+                    f"Invalid token transport header: {system.settings.chroma_auth_token_transport_header}. "
+                    f"Must be one of {allowed_token_headers}"
+                )
```

Comment:
This took me a couple reads to understand. Could we please directly check against the two possible values instead? I like having a better error message, I just want future engineers to easily be able to understand what this is doing.

Reply:
Done